src/azure-cli/azure/cli/command_modules/vm/custom.py

```diff
@@ -3044,7 +3054,13 @@ def create_image_version(cmd, resource_group_name, gallery_name, gallery_image_n
     ImageVersionPublishingProfile, GalleryArtifactSource, ManagedArtifact, ImageVersion, TargetRegion = cmd.get_models(
         'GalleryImageVersionPublishingProfile', 'GalleryArtifactSource', 'ManagedArtifact', 'GalleryImageVersion',
         'TargetRegion')
-    client = _compute_client_factory(cmd.cli_ctx)
+    aux_subscriptions = None
```

Comment:
could we reuse this part through validator?

Reply:
Good suggestion. Add a `_parse_aux_subscriptions` method now.